app/src/main/res/layout/item_referral_banner.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
```

Comment:
you sure you can't make ConstraintLayout as root? and add some margins or spacer